### ISSUE TITLE
HELP - Cards are not being created

### ISSUE THREAD

**Issue by JPsychotic** [CollinHeist/TitleCardMaker#383]
### Installation

Docker - develop tag

### Describe your Problem

Hey, sorry to bother again but my setup stopped working :( I reorganized my config files and tried executing TCM again but it did not create cards at all. For debugging purposes I removed all but one show from my series.yml but it did not create cards for it. I removed the source directory before executing TCM so it has to pull the images from jellyfin. The images from jellyfin are stored in the source directory but not as cards. They look the same as before :( The log doesn't say anything either...

Greetings and thanks in advance for helping!

### Preference File

```yaml
options:
  source: ./source/
  series:
  - ./config/animes.yml
  card_type: anime
  card_extension: png
  card_dimensions: 1920x1080
  validate_fonts: true
  sync_specials: false
  episode_data_source: jellyfin
  image_source_priority: jellyfin
  language_codes:
  - de
  - en
jellyfin:
  url: https://jellyfin.local/
  api_key: api_key
  username: Jeremy
  verify_ssl: false
  watched_style: unique
  unwatched_style: unique
```

### Series YAML

```yaml
libraries:
  Serien:
    media_server: jellyfin
    path: D:\data\tvshows
series:
  Jujutsu Kaisen (2020): {library: Serien, media_directory: 'D:\data\tvshows\Jujutsu Kaisen', card_type: anime, extras: {kanji: 呪術廻戦, require_kanji: true}}
```

### Debug Log

[DEBUG] [08-14-23 16:08:24.438] Runtime arguments  :
[DEBUG] [08-14-23 16:08:24.438]        preferences : D:\SourceCode\TitleCardMaker\preferences.yml
[DEBUG] [08-14-23 16:08:24.438]                run : True
[DEBUG] [08-14-23 16:08:24.438]               sync : False
[DEBUG] [08-14-23 16:08:24.438]          frequency : {'interval': 12, 'unit': 'hours'}
[DEBUG] [08-14-23 16:08:24.438]            missing : D:\SourceCode\TitleCardMaker\missing.yml
[DEBUG] [08-14-23 16:08:24.438]                log : INFO
[DEBUG] [08-14-23 16:08:24.438]           no_color : False
[DEBUG] [08-14-23 16:08:24.438] tautulli_frequency : {'interval': 4, 'unit': 'minutes'}
[INFO] [08-14-23 16:08:24.444] Read preference file "D:\SourceCode\TitleCardMaker\preferences.yml"
[DEBUG] [08-14-23 16:08:24.487] Using "magick " ImageMagick command prefix
[DEBUG] [08-14-23 16:08:24.496] Deleted show record database, was version v1.14.2-develop20
[INFO] [08-14-23 16:08:24.496] Read 0 show records
[INFO] [08-14-23 16:08:24.497] Starting TitleCardMaker (v1.14.2-develop24)
[INFO] [08-14-23 16:08:24.724] New version of TitleCardMaker (v1.14.2) available
[INFO] [08-14-23 16:08:24.724] Get the latest version with "git pull origin"
[INFO] [08-14-23 16:08:24.727] Read preference file "D:\SourceCode\TitleCardMaker\preferences.yml"
[DEBUG] [08-14-23 16:08:24.753] Using "magick " ImageMagick command prefix
[DEBUG] [08-14-23 16:08:24.757] Reset PersistentDatabase[remote_assets.json]
[DEBUG] [08-14-23 16:08:24.758] Not verifying SSL connections for Jellyfin
[INFO] [08-14-23 16:08:24.790] Starting to sync to series YAML files..
[INFO] [08-14-23 16:08:24.791] Reading series YAML file "D:\SourceCode\TitleCardMaker\config\animes.yml"..
[INFO] [08-14-23 16:08:24.823] Added 29 episodes to "Jujutsu Kaisen (2020)"
[DEBUG] [08-14-23 16:08:24.857] Downloaded s1e1.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.870] Downloaded s1e2.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.878] Downloaded s1e3.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.886] Downloaded s1e4.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.897] Downloaded s1e5.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.906] Downloaded s1e6.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.915] Downloaded s1e7.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.923] Downloaded s1e8.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.933] Downloaded s1e9.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.940] Downloaded s1e10.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.948] Downloaded s1e11.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.955] Downloaded s1e12.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.964] Downloaded s1e13.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.972] Downloaded s1e14.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.984] Downloaded s1e15.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.992] Downloaded s1e16.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:24.999] Downloaded s1e17.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.006] Downloaded s1e18.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.014] Downloaded s1e19.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.022] Downloaded s1e20.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.031] Downloaded s1e21.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.040] Downloaded s1e22.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.046] Downloaded s1e23.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.056] Downloaded s1e24.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.063] Downloaded s2e1.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.070] Downloaded s2e2.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.077] Downloaded s2e3.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.085] Downloaded s2e4.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[DEBUG] [08-14-23 16:08:25.091] Downloaded s2e5.jpg for "Jujutsu Kaisen (2020)" from jellyfin
[INFO] [08-14-23 16:08:25.106] Reading series YAML file "D:\SourceCode\TitleCardMaker\config\animes.yml"..
[INFO] [08-14-23 16:08:25.123] Wrote missing assets to "D:\SourceCode\TitleCardMaker\missing.yml"

**Comment by CollinHeist** [comment]
@JPsychotic It's no bother at all!

Could you check if there are there Cards already created at `D:\data\tvshows\Jujutsu Kaisen`? The logs are exactly identical to what would occur if you deleted the source folder but the cards already existed.

**Comment by JPsychotic** [comment]
I feel so dumb now. It's working again after deleting the `D:\data\tvshows\Jujutsu Kaisen` (cards were in there...). I got so confused because I ran it with card_type frame for the first time and after that I deleted the source directory because I wanted the anime card_type. After that the images in the source directory had card_type frame and because of that I thought the created cards are stored there... but thanks for helping me again :)
